src/derivatives/05-diffusion-derivatives.md

```diff
@@ -517,38 +506,37 @@ another.
                                  [ 128.8 ] ],
         "SphericalHarmonicDegree": 0,
         "Tissue": "Gray matter"
+        },
+        "csf":
+        {
+            XXX
+        }
     }
     ```
 
+
 -   An FSL `bedpostx` Ball-And-Sticks fit (including both mean parameters and
     bootstrap realisations):
 
     ```Text
     my_diffusion_pipeline/
         sub-01/
             dwi/
-                sub-01_model-bs_param-bzero_model.nii.gz
-                sub-01_model-bs_param-bzero_model.json
-                sub-01_model-bs_param-md_mdp.nii.gz
-                sub-01_model-bs_param-md_mdp.json
-                sub-01_model-bs_param-stdd_mdp.nii.gz
-                sub-01_model-bs_param-stdd_mdp.json
-                sub-01_model-bs_param-sticks_mdp.nii.gz
-                sub-01_model-bs_param-sticks_mdp.json
-                sub-01_model-bs_param-sticks_model.nii.gz
-                sub-01_model-bs_param-sticks_model.json
-                sub-01_model-bs_model.json
+                sub-01_model-bs_dwimap.json
+                sub-01_model-bs_param-bzero_dwimap.nii.gz
+                sub-01_model-bs_param-md_dwimap.nii.gz
+                sub-01_model-bs_param-stdd_dwimap.nii.gz
+                sub-01_model-bs_param-sticks_dwimap.nii.gz
+                sub-01_model-bs_param-sticksboostrap_dwimap.nii.gz
     ```
 
     Dimensions of NIfTI image "`sub-01_model-bs_param-bzero_model.nii.gz`": *I*x*J*x*K* ([scalar](#data-scalar))
-    Dimensions of NIfTI image "`sub-01_model-bs_param-ms_mdp.nii.gz`": *I*x*J*x*K* ([scalar](#data-scalar))
-    Dimensions of NIfTI image "`sub-01_model-bs_param-stdd_mdp.nii.gz`": *I*x*J*x*K* ([scalar](#data-scalar))
-    Dimensions of NIfTI image "`sub-01_model-bs_param-sticks_mdp.nii.gz`": *I*x*J*x*K*x9 ([spherical coordinates](#data-spherical), distance from origin encodes fibre volume fraction)
-    Dimensions of NIfTI image "`sub-01_model-bs_param-sticks_model.nii.gz`": *I*x*J*x*K*x9x50 ([spherical coordinates](#data-spherical), distance from origin encodes fibre volume fraction; 50 bootstrap realisations)
+    Dimensions of NIfTI image "`sub-01_model-bs_param-md_dwimap.nii.gz`": *I*x*J*x*K* ([scalar](#data-scalar))
+    Dimensions of NIfTI image "`sub-01_model-bs_param-stdd_dwimap.nii.gz`": *I*x*J*x*K* ([scalar](#data-scalar))
+    Dimensions of NIfTI image "`sub-01_model-bs_param-sticks_dwimap.nii.gz`": *I*x*J*x*K*x9 ([spherical coordinates](#data-spherical), distance from origin encodes fibre volume fraction)
+    Dimensions of NIfTI image "`sub-01_model-bs_param-sticksboostrap_dwimap.nii.gz`": *I*x*J*x*K*x9x50 ([spherical coordinates](#data-spherical), distance from origin encodes fibre volume fraction; 50 bootstrap realisations)
```

Comment:
Dealing with bootstrapped vs. aggregated model fits probably needs to be re-thought in the context of this PR; I don't think it's as simple as taking the prior example and renaming files.

Where there is a distinction between model fit and model-derived parameters, one utility of that distinction is that it is wholly applicable to the separation of bootstrap realisations from the aggregate across realisations. The same software command may yield both, but that's just a detail of the current state of the interface of that specific software command. From a data processing perspective, what is yielded from the process of fitting the model to the empirical data is the bootstrap realisations. Once you have those, you can freely compute statistics across realisations; reference to the empirical image data is no longer required.

In the absence of that distinction, you need some other way of distinguishing between the individual realisations and the aggregate. Trouble is, the "parameter" of the model being encoded is exactly the same; it's a question of whether you have multiple instances or just one. Therefore I'm not a fan of that distinction appearing as part of the "`param`" entity. I had previously in #61 played with the prospect of using a "`_stat-`" entity to express the fact that there's some aggregate statistic being computed across some data dimension; in this scenario it's not a trivial "average the values in the image", it's computing an average orientation in S2 across bootstrap realisations, but the concept still stands. Naively it feels to me a more appropriate place to encode / disambiguate this information, but conversely it potentially has other consequences in terms of eg. expressing not only the statistic that was computed but along which dimension, and then it starts to look more like provenance. Another alternative would be to have the bootstrap realisations vs. mean across bootstraps as different "`_model-`"'s, though I'm unsurprisingly not a terribly big fan of that either. I suspect that at the time I found the MDP vs. MFP distinction to be the most functional here, even if it doesn't conform to people's preconceptions of what these data are.

Reply:
I would rather leave this up to the researchers using these files. Presumably they need to understand the difference between bootstrap samples and summaries across these samples if they are using this model, and they can choose how they would like to handle any of these issues. We might want to clarify in the metadata how the summarization (into `sticks`) happened (median bs? mean bs? A run on the intact sample?).

Reply:
Deferred thread to #38.